signer/migrations/0019__p2p_peers.sql

```diff
@@ -0,0 +1,20 @@
+-- Stores information about known libp2p peers.
+CREATE TABLE p2p_peers (
+    -- The libp2p PeerId of the peer (base58 encoded multihash).
+    peer_id TEXT NOT NULL,
+    -- The public key of the peer
+    public_key BYTEA NOT NULL,
+    -- The last known reachable multiaddress for this peer.
+    address TEXT NOT NULL,
```

Comment:
Are we sure that we want to serialize the `peer_id` and `address` as strings? Having once been bitten by using strings instead of bytes, I tend to be cautious and recommend that we use bytea if that is the underlying representation, which appears to be the case for `Multiaddr`, less so for `peer_id` but I'd expect that to be more stable than their string representation.

Reply:
Yeah, I had the same exact reasoning for storing the strings when I was bitten (the case was storing stacks addresses as strings, we have a ticket briefly describing the issue in https://github.com/stacks-sbtc/sbtc/issues/1231).

Reply:
Not sure what to do, just sayin we should think again